src/lib/reasoners/sat_solver_sig.mli

```diff
@@ -128,17 +134,38 @@ module type S = sig
   (** [get_unknown_reason t] returns the reason Alt-Ergo raised
       [I_dont_know] if it did. If it did not, returns [None]. *)
 
-  val get_value : t -> Expr.t -> Expr.t option
-  (** [get_value t e] returns the value of [e] as a constant expression
-      in the current model generated. Returns [None] if can't decide. *)
-
   val get_objectives : t -> Objective.Model.t option
   (** [get_objectives t] returns the current objective values.
 
       @return [None] if there is no objective or the environment is
       unsatisfiable. *)
 end
 
+type lbool = True | False | Unknown
+
+val pp_lbool : lbool Fmt.t
+
+module type S = sig
+  include Internal
```

Comment:
I know this PR will add an extra layer in this millefeuille but for me the main purpose of `Frontend` is to parse declarations from `solving_loop`. I agree to use first-class module to avoid the huge re-indentation.